core/src/main/java/jenkins/management/StatisticsLink.java

```diff
@@ -48,6 +52,12 @@ public String getDescription() {
         return Messages.StatisticsLink_Description();
     }
 
+    @CheckForNull
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
+    }
+
```

Comment:
No special permissions are needed to access this anyway: https://ci.jenkins.io/load-statistics